src/main/java/gtPlusPlus/xmod/gregtech/common/helpers/ChargingHelper.java

```diff
@@ -384,6 +388,18 @@ else if ((mitemCurrentCharge + (mVoltageIncrease*5)) <= (mItemMaxCharge - (mVolt
 
 				}
 			}
+			else if (isItemValidRF(mTemp)) {
+				try {
+					IEnergyContainerItem rfItem = (IEnergyContainerItem)mTemp.getItem();
+					long RF = Math.min(rfItem.getMaxEnergyStored(mTemp) - rfItem.getEnergyStored(mTemp), mEntity.getEUVar() * mEUtoRF / 100L);
+					RF = rfItem.receiveEnergy(mTemp, RF > Integer.MAX_VALUE ? Integer.MAX_VALUE : (int) RF, false);
+					RF = RF * 100L / mEUtoRF;
+					mEntity.setEUVar(Math.max(mEntity.getEUVar() - RF, 0));
+					return RF;
```

Comment:
Don't return, like charging EU does. Also insert `mChargedItems++`, or remove lines below, they're not needed
```
if (mChargedItems < 1){
	return mEuStoredOriginal;
}
```